.ci-builds/.azure-pipelines.yml

```diff
@@ -628,45 +628,52 @@ jobs:
+      displayName: 'Build property data export files'
+      env:
+        LD_LIBRARY_PATH: lib
+    - script: |
         cd icu4c/source
-        LD_LIBRARY_PATH=lib ./bin/icuexportdata --mode uprops --index --copyright --verbose --destdir icuexportdata_uprops_sample/fast --trie-type fast AHex gc nt Basic_Emoji sc WSpace blank
-        LD_LIBRARY_PATH=lib ./bin/icuexportdata --mode uprops --index --copyright --verbose --destdir icuexportdata_uprops_sample/small --trie-type small AHex gc nt Basic_Emoji sc WSpace blank
-      displayName: 'Build Unicode property data export file (Sample)'
+        mkdir -p icuexport/norm/fast
+        ./bin/icuexportdata --mode norm --index --copyright --verbose --destdir icuexport/norm/fast --trie-type fast --all
+        mkdir -p icuexport/norm/small
+        ./bin/icuexportdata --mode norm --index --copyright --verbose --destdir icuexport/norm/small --trie-type small --all
+      displayName: 'Build normalization data export files'
+      env:
+        LD_LIBRARY_PATH: lib
     - script: |
-        mkdir -p icu4c/source/icuexportdata_uprops_full/collation_unihan
-        mkdir -p icu4c/source/icuexportdata_uprops_full/collation_implicithan
         cd icu4c/source
-        cd data/coll
-        FILES=`echo *.txt`
-        cd -
-        LD_LIBRARY_PATH=lib ./bin/genrb -X -s data/coll/ -d icuexportdata_uprops_full/collation_unihan --ucadata data/in/coll/ucadata-unihan-icu4x.icu $FILES
-        LD_LIBRARY_PATH=lib ./bin/genrb -X -s data/coll/ -d icuexportdata_uprops_full/collation_implicithan --ucadata data/in/coll/ucadata-implicithan-icu4x.icu $FILES
-        rm icuexportdata_uprops_full/collation_unihan/*.res
-        rm icuexportdata_uprops_full/collation_implicithan/*.res
-      displayName: 'Build collation data export file'
-    - task: PublishBuildArtifacts@1
-      displayName: 'Publish Artifact: icuexportdata_uprops_full'
+        mkdir -p icuexport/ucase/fast
+        ./bin/icuexportdata --mode ucase --index --copyright --verbose --destdir icuexport/ucase/fast --trie-type fast --all
+        mkdir -p icuexport/ucase/small
+        ./bin/icuexportdata --mode ucase --index --copyright --verbose --destdir icuexport/ucase/small --trie-type small --all
+      displayName: 'Build case data export files'
+      env:
+        LD_LIBRARY_PATH: lib
+    - script: |
+        cd icu4c/source
+        FILES=`ls data/coll "*.txt"` 
+        mkdir -p icuexport/collation/unihan
+        ./bin/genrb -X -s data/coll/ --ucadata data/in/coll/ucadata-unihan-icu4x.icu -d icuexport/collation/unihan $FILES
+        rm icuexport/collation/unihan/*.res
+        mkdir -p icuexport/collation/implicithan
+        ./bin/genrb -X -s data/coll/ --ucadata data/in/coll/ucadata-implicithan-icu4x.icu -d icuexport/collation/implicithan $FILES
+        rm icuexport/collation/implicithan/*.res
+      displayName: 'Build collation data export files'
+      env:
+        LD_LIBRARY_PATH: lib
+    - task: ArchiveFiles@2
```

Comment:
Issue: I think this results in a double-zip. Check the zip file from here:

https://dev.azure.com/ms/icu/_build/results?buildId=325613&view=artifacts&pathAsName=false&type=publishedArtifacts

I recommend you avoid double-zipping, but if you want people to unzip the outer zip file as part of uploading the artifact, you should update the instructions, which are located in */docs/processes/release/tasks/publish/index.md*. Keep in mind that humans do these steps, don't know what they are necessarily doing, and are likely to make mistakes, so the fewer steps the better.

Reply:
ok, if you don't want the root folder included, then I agree, this seems like the right way.

My suggestion therefore becomes: Make the name of the inner zip file different from the name of the outer zip file